csm/conf/setup.py

```diff
@@ -774,6 +732,7 @@ def config(self, args):
             self._verify_args(args)
             if not self._replacement_node_flag:
                 self.Config.create(args)
+                UDSConfigGenerator.apply()
```

Comment:
How will uds and haproxy config will be restored in case of node replacement?
@pawankumarsrivastava Please review.

Reply:
In retrospect, I think it makes sense to move it out of this conditional branch. I will wait for Pawan to confirm.

Reply:
@ajaysrivas  in case of node replacement csm_setup commands runs again.. so it should restore it again

Reply:
I have updated the code. `UDSConfigGenerator.apply()` is now out of this conditional branch.